files/build_templates/sonic_debian_extension.j2

```diff
@@ -401,7 +401,7 @@ sudo LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS ta
 sudo LANG=C chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS tag {{imagename}}:latest {{imagebasename}}:latest
 {% endif %}
 {% endfor %}
-sudo umount $FILESYSTEM_ROOT/target
+sudo LANG=C chroot $FILESYSTEM_ROOT umount target
```

Comment:
>chroot [](start = 12, length = 6)

It is mounted without chroot
```
sudo mount --bind target $FILESYSTEM_ROOT/target
```
Why umount with chroot?

Reply:
check the failed here. https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-vs-image-buster/43/console

```
06:58:13.578  + sudo umount ./fsroot/target
06:58:13.578  umount: ./fsroot/target: umount failed: No such file or directory.
```

I am not sure why it failed in the first time.

Reply:
let me add some more debug information to understand why

Reply:
it looks like it does not have mtab file

```
+ sudo LANG=C chroot ./fsroot docker tag docker-syncd-vs:latest docker-syncd-vs:buster.0-dirty-20200409.090820
+ sudo mount
mount: failed to read mtab: No such file or directory
```

Reply:
the problem is /proc is not not mount. Now, I mount /proc in sonic-slave docker and now umount outside fsroot is ok now.